### PR TITLE
Backport PR #2967 to 2.8

### PR DESCRIPTION
**Pull request by DavidKang**

### CODE REVIEW

File: ReleaseNotes-2.8.1 [hunk 1]
```diff
@@ -19,4 +19,4 @@ Changes:
 Bugfixes:
 =========
 
-* none
+* [webui] Errors handling of MissingParameterError exception in webui for live build log
```
Comment (bgeuken) [inline ReleaseNotes-2.8.1]: Could you rephrase this to a real sentence?:-)

Maybe "Fixes bug in live build log when no architecture or repository parameter was given" (or when it was invalid)

**Comment by DavidKang** [comment]
@hennevogel, please re-review it :smiley_cat:
